diff --git a/dist/index.js b/dist/index.js
index a76837352821c72266ae7fafa3690e311efe2a76..af3585abff6d320795b867562bcea7a444af94a5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -49,9 +49,9 @@ exports.isValidPath = (path) => {
         .some(isNaN);
 };
 exports.derivePath = (path, seed, offset = HARDENED_OFFSET) => {
-    if (!exports.isValidPath(path)) {
-        throw new Error('Invalid derivation path');
-    }
+    // if (!exports.isValidPath(path)) {
+    //     throw new Error('Invalid derivation path');
+    // }
     const { key, chainCode } = exports.getMasterKeyFromSeed(seed);
     const segments = path
         .split('/')
